packages/derive/src/callable_points.rs

```diff
@@ -0,0 +1,52 @@
+use proc_macro2::{TokenStream, TokenTree};
+use quote::quote;
+use syn::Item;
+
+use crate::callable_point;
+
+pub fn make_callable_points(body: Vec<Item>) -> (Vec<TokenTree>, Vec<TokenStream>) {
+    let mut list_callable_points = Vec::new();
+    let mut maked = Vec::new();
+
+    for i in &body {
+        if let syn::Item::Fn(function) = i.clone() {
+            let (function_remove_macro, is_strip) = strip_callable_point(function.clone());
```

Comment:
can `strip_callable_point` take `&ItemFn` and return `ItemFn`?

Reply:
why?